### PR TITLE
Add Redis Consumer service plugin.

### PR DESCRIPTION
**Pull request by OrangeTux**
This PR implements Redis Consumer plugin as discussed in #759 . Tests and README are missing at the moment but I want some feedback if I'm heading in the right direction.

### CODE REVIEW

File: plugins/inputs/redis_consumer/redis_consumer.go [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  ##  [protocol://][:password]@address[:port]
+  ##  e.g.
+  ##    tcp://localhost:6379
+  ##    tcp://:password@192.168.99.100
+  ##
+  ## If no servers are specified, then localhost is used as the host.
+  ## If no port is specified, 6379 is used
+  servers = ["tcp://localhost:6379"]
+
+  ##  List of channels to listen to. Selecting channels using pattern-matching
```
Comment (sparrc) [inline plugins/inputs/redis_consumer/redis_consumer.go]: what type of pattern-matching? Is it regex or something redis-specific? Can you give a short example?

Comment (OrangeTux) [inline plugins/inputs/redis_consumer/redis_consumer.go]: Clarified with new commit.

**Comment by sparrc** [comment]
thanks @OrangeTux, overall this looks good but you'll need to:

1. rebase your changes off of HEAD,
2. run `go vet ./...` and `go fmt ./...` on your code
3. finish the README
4. write unit tests

**Comment by sparrc** [comment]
Thank you for the contribution @OrangeTux, I'm closing this for inactivity but feel free to open a new PR when it's ready

**Comment by soulmachine** [comment]
@OrangeTux Any updates about this plugin? Really need it

**Comment by OrangeTux** [comment]
@soulmachine I'm sorry. I haven't been using Telegraf and this plugin for years now. You can take my code and add the requested changes if you want.

**Pull request closed without merging**